k8s/openebs-operator.yaml

```diff
@@ -525,7 +530,7 @@ spec:
           value: "10737418240"
         # Specify the number of sparse files to be created
         - name: SPARSE_FILE_COUNT
-          value: "1"
+          value: "0"
```

Comment:
changing to '0' will break the CI in maya and other repos, let it be '1'.  We are mantaining the count '0' in charts and release yamls that would be good enough.

Reply:
Done.